utility/src/main/java/org/oppia/android/util/parser/html/CustomHtmlContentHandler.kt

```diff
@@ -185,6 +187,21 @@ class CustomHtmlContentHandler private constructor(
      * @param indentation The zero-based indentation level of this item.
      */
     fun handleClosingTag(output: Editable, indentation: Int, tag: String) {}
+
+    /**
+     * Called when a custom tag is encountered. This is always called after the closing tag.
+     *
+     * @param attributes the tag's attributes
+     * @param openIndex the index in the output [Editable] at which this tag begins
+     * @param closeIndex the index in the output [Editable] at which this tag ends
+     * @param output the destination [Editable] to which spans can be added
+     */
+    fun handleContentDescription(
```

Comment:
In terms of names, 'handle content description' doesn't make a lot of sense to me.

But rather than considering renaming this, what's the intent for having a separate method vs. just including this logic in one of the other overrides? It seems that special content descriptions are only needed for images, so I don't see what value a separate method provides.